charts/rancher-vsphere-cpi/rancher-vsphere-cpi/100.0.0/templates/vsphere-cpi-rbac.yaml

```diff
@@ -1,14 +1,15 @@
+{{- if .Values.cloudControllerManager.rbac.enabled -}}
 # Source: https://github.com/kubernetes/cloud-provider-vsphere/blob/master/releases/v1.19/vsphere-cloud-controller-manager.yaml
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
```

Comment:
Using `vsphere-` seems fine, but my preference would be to using `{{ .Chart.Name }}` so that we don't have conflicts with any other vsphere resources (e.g. will this potentially conflict with `rancher-vsphere-csi` resources?)

What are your thoughts?

Reply:
I don't think it will conflict because they are supposed to be installed together and we probably would've ran into problems by now with the more generic name. I tried `{{ .Chart.Name }}` and `{{ .Release.Name }}` and they end up with super long names so I figured adding vsphere was enough, but I'll go ahead and update it to `{{ .Chart.Name }}` to be safe.